x-pack/plugins/security_solution/common/entity_analytics/asset_criticality/constants.ts

```diff
@@ -5,10 +5,18 @@
  * 2.0.
  */
 
-export const ASSET_CRITICALITY_URL = `/internal/asset_criticality` as const;
-export const ASSET_CRITICALITY_PRIVILEGES_URL = `${ASSET_CRITICALITY_URL}/privileges` as const;
-export const ASSET_CRITICALITY_STATUS_URL = `${ASSET_CRITICALITY_URL}/status` as const;
-export const ASSET_CRITICALITY_CSV_UPLOAD_URL = `${ASSET_CRITICALITY_URL}/upload_csv` as const;
+export const ASSET_CRITICALITY_INTERNAL_URL = `/internal/asset_criticality` as const;
+export const ASSET_CRITICALITY_INTERNAL_PRIVILEGES_URL =
+  `${ASSET_CRITICALITY_INTERNAL_URL}/privileges` as const;
+export const ASSET_CRITICALITY_INTERNAL_STATUS_URL =
+  `${ASSET_CRITICALITY_INTERNAL_URL}/status` as const;
+export const ASSET_CRITICALITY_INTERNAL_CSV_UPLOAD_URL =
+  `${ASSET_CRITICALITY_INTERNAL_URL}/upload_csv` as const;
+
+// TODO: rename this
```

Comment:
Is this leftover stuff or should we actually rename it? why?

Reply:
Ah this is leftover from dev thanks for spottinh!